sway-book/src/documentation/language/functions/associated-function.md

```diff
@@ -0,0 +1,27 @@
+# Associated Functions
+
+Associated functions are similar to methods in that they are also defined in the context of a [struct](../built-ins/structs.md) or [enum](../built-ins/enums.md), but they do not use any of the data in the struct and as a result do not take `self` as a parameter.
+
+Associated functions could be standalone functions, but they are included in a specific type for organizational or semantic reasons.
+
+## Decleration
+
+In this example we will take a look at a struct however an enum will work in the same way.
```

Comment:
```suggestion
In this example, we will take a look at a struct. Note that an enum will work in the same way.
```

Reply:
This breaks the flow of how the sentence is read. It becomes more of like a bullet point by bullet point explanation rather than a sentence.

Reply:
technically speaking, if you want this to be one sentence, it must be :

> In this example, we will take a look at a struct; however, an enum will work in the same way.

That said, I do not think any of the versions discussed will make or break the readability of the sentence. 🤷